lib/src/model/product_image.dart

```diff
@@ -130,6 +130,7 @@ class ProductImage {
     required String this.imgid,
     this.width,
     this.height,
+    this.uploaded,
```

Comment:
I think we can have a better variable name, maybe something like uploaded_on or created_at. Or is it the same var name in other places?

Reply:
Hi @AshAman999!

In this PR we have an `'uploaded_t'` JSON tag and an `int? uploaded` field.

If we compare to similar fields:
* we also have a `'created_t'` JSON tag and a `created` field (cf. in `Product`)
* but instead of `int?` in this PR, we do have a much more explicit `DateTime?`

Would you be OK with `DateTime? uploaded`? That would be consistent with the rest of the timestamp fields in the package, and wouldn't be confusing with a `DateTime?` type.